.github/workflows/im-reusable-finish-build-workflow.yml

```diff
@@ -0,0 +1,136 @@
+# The purpose of this reusable job is to run the final steps of a CI build workflow
+# which includes posting a status to teams, updating a PR comment and setting the final 
+# workflow outcome.
+
+name: Finish Workflow
+on:
+  workflow_call:
+    inputs:
+      next-version:
+        description: The next generated version
+        type: string
+        required: false
+        default: 'N/A'
+      title-of-teams-post:
+        description: 'Title of the Teams post that reports the build status.'
+        required: true
+        type: string
+      is-merge-to-main:
+          description: Flag indicating whether this is a merge to the default branch.
+          required: true
+          type: string
```

Comment:
I initially had this as a bool, but when I used an output from my examine-triggers job as the input it failed because that output was a string, not a bool.  It is easier to change this arg to a string type and then do the proper conversion in the github-script below where it is utilized.